content/docs/tutorials/istio-csr/istio-csr.md

```diff
@@ -265,4 +273,4 @@ namespace or application.
 Assuming your running inside kind, you can simply remove the cluster:
 
 ```shell
-kind delete cluster kind
+kind delete cluster
```

Comment:
My first instinct was to change this to `kind delete clusters kind` or `kind delete cluster --name kind` but I see our setup instructions aren't explicit about the name so I don't think our tear-down instructions need to be either. Good change, thanks!